spec/models/ability_spec.rb

```diff
@@ -10,6 +10,7 @@
     it { is_expected.not_to be_able_to(:create, ContentBlock) }
     it { is_expected.not_to be_able_to(:update, ContentBlock) }
     it { is_expected.to     be_able_to(:read,  ContentBlock) }
+    it { is_expected.to     be_able_to(:view_upload, GenericFile) }
```

Comment:
> Why view_upload here?

This is actually a permission to hide the "Share Your Work" button on the home page. Maybe a different name? :view_share_work ?